vizro-core/src/vizro/static/css/layout.css

```diff
@@ -182,10 +183,23 @@ div.dashboard_container .custom-tooltip {
 
 #page-main {
   display: flex;
+  flex: 1 1 0;
   flex-direction: row;
   height: calc(100% - 64px);
 }
 
 #dashboard-title {
   margin: 0;
 }
+
+#logo {
+  height: 100%;
+}
+
+#logo-and-title {
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  gap: 12px;
```

Comment:
Should we maybe increase the gap between the logo and the title? It looks a bit off as the logo does not align with the nav bar below. 

<img width="373" alt="Screenshot 2024-01-16 at 15 28 48" src="https://github.com/mckinsey/vizro/assets/108531476/d3e55a65-d6f3-4e6f-9b99-88051208de30">

Increasing it to 16px leads to better alignment with the navbar. 
<img width="363" alt="Screenshot 2024-01-16 at 15 27 46" src="https://github.com/mckinsey/vizro/assets/108531476/2154668a-17f2-49d5-9048-ac4d77471235">

Reply:
I like that proposal! Will implement :)